.github/workflows/pull-request.yaml

```diff
@@ -1,38 +1,47 @@
 # This workflow will lint the PR's title and commit message.
-# https://github.com/CondeNast/conventional-pull-request-action
-# https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md
-# https://github.com/CondeNast/conventional-pull-request-action/issues/15
 
 name: Pull Request
 on:
   pull_request:
     branches: [main, staging]
     types:
     - opened
+    - reopened
     - edited
     - synchronize
```

Comment:
I also noticed that we’re using two different notations for [YAML block collections](https://yaml.org/spec/1.1/current.html#id857181) (in this case sequences):
```yaml
branches: [main, staging]
```
is what the spec calls a [“flow style”](https://yaml.org/spec/1.1/current.html#flow%20style/information%20model) equivalent to writing out the sequence itself in “block style”:
```yaml
types:
- opened
- reopened
- edited
- synchronize
```
Should we agree on one consistent notation?

Reply:
> Should we agree on one consistent notation?

Sure, I'm happy with either.

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/263/commits/a6103832b49ab416cfa51398fa68d22243fd1d90: for now I switched to “block style”.